src/components/books/list.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable no-unused-vars */
+import Books from './books.js';
+import Form from './form.js';
+import './list.css';
+
+function List() {
```

Comment:
- [ ] Please, use ES6 best practices, in this case, this could improve being an arrow function. 🦊